domains.md

```diff
@@ -31,6 +31,7 @@
 | airhornbot.com            | ^                                                                   |
 | bigbeans.solutions        | [April Fools 2017](https://youtu.be/9Z4GW6Vd6NI)                    |
 | watchanimeattheoffice.com | HypeSquad form placeholder/meme                                     |
+| nicememe.website          | [Redirect URL Example](https://discord.com/developers/docs/topics/oauth2#authorization-code-grant-redirect-url-example) |
```

Comment:
I'm fairly sure that this domain isn't owned by Discord since it was registered before Discord was even created.
![image](https://user-images.githubusercontent.com/52178694/147618230-f2a8ec8f-d6a7-429f-a07f-c61464200bc4.png)